librespot-android-sink/src/main/java/xyz/gianlu/librespot/android/sink/AndroidSinkOutput.java

```diff
@@ -19,24 +19,51 @@
     private AudioTrack track;
     private float lastVolume = -1;
 
-    public AndroidSinkOutput() {
-    }
-
     @Override
     public boolean start(@NotNull OutputAudioFormat format) throws SinkException {
-        AudioTrack.Builder builder = new AudioTrack.Builder();
-        builder.setAudioFormat(new AudioFormat.Builder()
-                .setSampleRate((int) format.getSampleRate())
-                .build());
+        int pcmEncoding = format.getSampleSizeInBits() == 16 ? AudioFormat.ENCODING_PCM_16BIT : AudioFormat.ENCODING_PCM_FLOAT;
```

Comment:
Do we expect anything but ENCODING_PCM_16BIT?

Reply:
ENCODING_PCM_16BIT is the only one we get and the only one that is actually supported in the mixer.